src/styles/index.scss

```diff
@@ -5,6 +5,7 @@
 // components
 @import "./components/button.scss";
 @import "./components/text-input.scss";
+@import "./components/cookieBanner.scss";
```

Comment:
the name the this file is not as consistent as the others.  Shall we agree on one style of sass naming?